qa/tasks/cephfs/kernel_mount.py

```diff
@@ -188,21 +188,15 @@ def read_debug_file(self, filename):
         """
         Read the debug file "filename", return None if the file doesn't exist.
         """
-        debug_dir = self._find_debug_dir()
 
-        pyscript = dedent("""
-            import os
-
-            print(open(os.path.join("{debug_dir}", "{filename}")).read())
-            """).format(debug_dir=debug_dir, filename=filename)
+        path = os.path.join(self._find_debug_dir(), filename)
 
+        stdout = StringIO()
         stderr = StringIO()
         try:
-            output = self.client_remote.sh([
-                'sudo', 'python3', '-c', pyscript
-            ], stderr=stderr, timeout=(5*60))
-
-            return output
+            self.client_remote.run_shell_payload(f"sudo dd if={path}", timeout=(5*60),
```

Comment:
The `dd` usually will print extra info like: 

```
$ dd if=/tmp/myfile
sfkjl
0+1 records in
0+1 records out
6 bytes copied, 6.4198e-05 s, 93.5 kB/s
```
How about use `cat` instead ?

Reply:
Or add `status=none` for `dd` command ?

Reply:
That information goes to stderr, should not be a problem.